tensorflow/compiler/xla/service/gpu/gemm_rewriter.cc

```diff
@@ -573,11 +585,26 @@ class GemmRewriterVisitor : public DfsHloRewriteVisitor {
     }
 
     if (Match(instr,
-              m::AddAnyOrder(GemmOrCublasLtMatmul(&existing_gemm).WithOneUser(),
-                             m::Op(&bias).WithPredicate(is_not_broadcast)))) {
+              m::AddAnyOrder(
+                  GemmOrCublasLtMatmulMaybeF8(&existing_gemm).WithOneUser(),
+                  m::Op(&bias).WithPredicate(is_not_broadcast)))) {
       return FuseMatrixBiasAdd(instr, bias, existing_gemm);
     }
 
+    HloInstruction *optional_bitcast_matrix = nullptr;
+    HloInstruction *optional_slice_matrix = nullptr;
+    if (Match(instr,
+              m::AddAnyOrder(
+                  OptionalBitcast(
+                      &optional_bitcast_matrix,
+                      OptionalSlice(
+                          &optional_slice_matrix,
+                          CublasLtMatmulMaybeF8(&existing_gemm).WithOneUser()))
+                      .WithOneUser(),
+                  m::Op(&bias).WithPredicate(is_not_broadcast)))) {
```

Comment:
This Match statement looks like it matches a superset of the previous Match statement. If that is the case, remove the previous match statement.

Reply:
Resolved.